RPIPs/RPIP-28.md

```diff
@@ -0,0 +1,47 @@
+---
+rpip: 28
+title: Deposits Under the Minimum
+description: Allow minipool deposits that include an appropriate RPL stake, even when under the "minimum" RPL threshold 
+author: Valdorff (@Valdorff)
+discussions-to: 
+status: Draft
+type: Protocol
+category: Core
+created: 2023-08-11
+---
+
+## Abstract
+
+Right now, if an NO falls below the minimum (value of RPL staked <10% of borrowed ETH), they can't 
+add a new minipool to that node unless they stake enough to reach the minimum AND cover the new
+minipool. Unless, of course, they make a new node. This means that we're creating an incentive to
+make Sybil nodes.
```

Comment:
side point: I would love if we could find alternative terminology than Sybil -- I realise it's an industry-wide thing, but it's loaded with stigma against mental illness that I cringe at participating in.

Reply:
"duplicate" would work for this RPIP although it doesn't capture the whole meaning maybe

Reply:
or "sock puppet node"

Reply:
sock puppet works for me